src/sentry/api/event_search.py

```diff
@@ -349,6 +365,54 @@ def convert_endpoint_params(params):
     ]
 
 
+def convert_search_filter_to_snuba_query(search_filter):
```

Comment:
This is mostly just moving conversion logic out of `get_snuba_query_args` so that it can be shared elsewhere. There are some other changes though